x-pack/test/defend_workflows_cypress/runner.ts

```diff
@@ -9,7 +9,44 @@ import { resolve } from 'path';
 import Url from 'url';
 import { withProcRunner } from '@kbn/dev-proc-runner';
 
+import { startRuntimeServices } from '@kbn/security-solution-plugin/scripts/endpoint/endpoint_agent_runner/runtime';
 import { FtrProviderContext } from './ftr_provider_context';
+import { AgentManager } from './agent';
+import { FleetManager } from './fleet_server';
+
+async function withFleetAgent(
+  { getService }: FtrProviderContext,
+  runner: (runnerEnv: Record<string, string>) => Promise<void>
+) {
+  const log = getService('log');
+  const config = getService('config');
+
+  const elasticUrl = Url.format(config.get('servers.elasticsearch'));
+  const kibanaUrl = Url.format(config.get('servers.kibana'));
+  const username = config.get('servers.elasticsearch.username');
+  const password = config.get('servers.elasticsearch.password');
+
+  await startRuntimeServices({
+    log,
+    elasticUrl,
+    kibanaUrl,
+    username,
+    password,
+    version: '8.7.0-SNAPSHOT',
```

Comment:
version hardcoded, because `8.8.0-SNAPSHOT` isn't available yet, we may want to update the service to try to use the matched version, but if it fails try to fall back to version-1?